integration-tests/ccip-tests/testconfig/tomls/ccip-crib.toml

```
@@ -140,6 +140,12 @@ Email = 'notreal@fakeemail.ch'
 Password = 'fj293fbBnlQ!f9vNs'
 InternalIP = 'app-node-5'
 
+[[CCIP.Env.ExistingCLCluster.NodeConfigs]]
+URL = 'https://skudasov-crib-ccip-node6.main.stage.cldev.sh/'
+Email = 'notreal@fakeemail.ch'
+Password = 'fj293fbBnlQ!f9vNs'
+InternalIP = 'app-node-6'
+
```

```suggestion
[[CCIP.Env.ExistingCLCluster.NodeConfigs]]
URL = 'https://skudasov-crib-ccip-node6.main.stage.cldev.sh/'
Email = 'notreal@fakeemail.ch'
Password = 'fj293fbBnlQ!f9vNs'
InternalIP = 'app-node-6'

```

@AnieeG @gheorghestrimtu do I add this back in?

---

Sounds like it is.